bugbug/models/invalid_compatibility_report.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from sklearn.compose import ColumnTransformer
+from sklearn.pipeline import Pipeline
+
+from bugbug import feature_cleanup, issue_features, utils
+from bugbug.model import IssueModel
+
+logger = logging.getLogger(__name__)
+
+
+class InvalidCompatibilityReportModel(IssueModel):
+    def __init__(self, lemmatization=False):
+        super().__init__(
+            owner="webcompat", repo="web-bugs", lemmatization=lemmatization
+        )
+
+        self.calculate_importance = False
+
+        feature_extractors = []
+
+        cleanup_functions = []
+
+        self.extraction_pipeline = Pipeline(
+            [
+                (
+                    "report_extractor",
+                    issue_features.IssueExtractor(
+                        feature_extractors, cleanup_functions, rollback=False
+                    ),
+                ),
+                (
+                    "union",
+                    ColumnTransformer(
+                        [
+                            (
+                                "first_comment",
+                                self.text_vectorizer(min_df=0.0001),
+                                "first_comment",
+                            ),
+                        ]
+                    ),
+                ),
+            ]
+        )
+
+        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+
+    def items_gen(self, classes):
+        # Do cleanup separately from extraction pipeline to
+        # make sure it's not applied during classification due to differences
+        # in text structure between GitHub issues and reports
+        cleanup_function = feature_cleanup.CleanCompatibilityReportDescription()
+
+        for issue in self.github.get_issues():
+            issue_number = issue["number"]
+            if issue_number not in classes:
+                continue
+
+            issue["body"] = cleanup_function(issue["body"])
+
+            yield issue, classes[issue_number]
```

Comment:
We could reuse `items_gen()` from the parent class:
```suggestion
        for issue, class in super().items_gen(classes):
            issue["body"] = cleanup_function(issue["body"])
            yield issue, class
```

Reply:
Ah true, thanks :)